testcases/init_token.sh.in

```diff
@@ -61,17 +61,17 @@ expect {
     timeout { send_user "Timeout during user PIN initialization\n"; exit 1 }
 }
 
-spawn @sbindir@/pkcsconf -c [lindex $argv 0] -p
+spawn env -u PKCS11_USER_PIN @sbindir@/pkcsconf -c [lindex $argv 0] -p
 expect {
-    "Enter user PIN: " { sleep .1; send "12345678\r"; }
+    "Enter the USER PIN: " { sleep .1; send "12345678\r"; }
     default { send_user "Error sending user PIN during user PIN setting\n"; exit 1 }
 }
 expect {
-    "Enter the new user PIN: " { sleep .1; send $env(PKCS11_USER_PIN); send "\r"; }
+    "Enter new USER PIN: " { sleep .1; send $env(PKCS11_USER_PIN); send "\r"; }
     default { send_user "Error sending new user PIN during user PIN setting\n"; exit 1 }
 }
 expect {
-    "Re-enter the new user PIN: " { sleep .1; send $env(PKCS11_USER_PIN); send "\r"; }
+    "Re-enter new USER PIN: " { sleep .1; send $env(PKCS11_USER_PIN); send "\r"; }
```

Comment:
You need to do the same in testcases/cleanup_vhsm.exp.in, testcases/init_token.sh.in, testcases/init_token.sh.in and testcases/init_vhsm.exp.in

Reply:
Ok, I'll check this and change it accordingly.

Reply:
Dito, not in the next version, but still on my to-do list.